cypress/support/elements/activity-page.js

```diff
@@ -11,5 +14,10 @@ class ActivityPage {
   getSidebarCloseButton() {
     return cy.get("[data-cy=sidebar-close-button");
   }
+  getSecondaryEmbeddable(type) { 
+    //types=["image-question","text-box","multiple-choice-question","open-response-question", 
+    //      "labbook-question", "iframe-interactive-question","image-video-interactive", ]
```

Comment:
is this obsolete?

Reply:
No, it's the way Kevin and I keep track of what the parameters should look like.

Reply:
Ah, got it!